libs/template/templates/default-sql/template/{{.project_name}}/scratch/exploration.ipynb.tmpl

```diff
@@ -0,0 +1,35 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "dc8c630c-1ea0-42e4-873f-e4dec4d3d416",
```

Comment:
Is this hard-coded nuid expected?

Reply:
It's part of the official serialization format that comes out of the workspace. So it's the same for any notebook that is committed to Git. As far as I know it's harmless and not really useful. But @kahing might know more about it.